### PR TITLE
Forgot consent in oidc table

### PR DESCRIPTION
**Pull request by mystiberry-okta**
## Description:
- Added prompt to table, improved description of prompt.

### Resolves:
[OKTA-162811](https://oktainc.atlassian.net/browse/OKTA-162811)

### CODE REVIEW

**Comment by mystiberry-okta** [comment]
@larsjohansen-okta , I wrecked the previous doc PR for this, so recreated it in a new doc PR. Here's the output--I'm pretty sure I got everything! Sorry for the hassle of not having the old comments

image

image

File: _source/_docs/api/resources/oidc.md [hunk 1]
```diff
@@ -87,13 +87,18 @@ of the callback response.
  * `idp`, `sessionToken` and `idp_scope` are Okta extensions to [the OpenID specification](http://openid.net/specs/openid-connect-core-1_0.html#Authentication).
     All other parameters comply with the OpenID Connect specification and their behavior is consistent with the specification.
 
-`prompt`:
+* `prompt`:
 
-There are three possible values for this parameter:
+    If no `prompt` parameter is specified, the standard behavior occurs:
+    * If an Okta session already exists, the user is silently authenticated. Otherwise, the user is prompted to authenticate.
+     * If scopes are requested that consent and consent is not yet given by the authenticated user, the user is prompted to give consent.
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oidc.md]: `If scopes are requested that consent and consent is not yet given` -> `If scopes are requested that requires consent and consent is not yet given`

File: _source/_docs/api/resources/oidc.md [hunk 2]
```diff
@@ -87,13 +87,18 @@ of the callback response.
...
+    There are four possible values for this parameter:
+
+    * `none`: Do not prompt for authentication or consent. If an Okta session already exists, the user is silently authenticated. Otherwise, an error is returned.
+    * `login`: Always prompt the user for authentication, regardless of whether they have an Okta session.
+    * `consent`: {% api_lifecycle ea %} Depending on the [values set for `consent_method` in the app and and `consent` on the scope](/docs/api/resources/apps.html#add-oauth-20-client-application), display the Okta consent dialog, even if the user has already given consent. User consent is available for Custom Authorization Servers (requires the API Access Management feature and the User Consent feature enabled).
+    * `login` `consent` or `consent` `login` (order doesn't matter): The user is always prompted for authentication, and the user consent dialog is displayed depending on the [values set for `consent_method` in the app and and `consent` on the scope](/docs/api/resources/apps.html#add-oauth-20-client-application), even if the user has already given consent.
```
Comment (larsjohansen-okta) [inline _source/_docs/api/resources/oidc.md]: `'login' 'consent' or 'consent' 'login'` -> `'login consent' or 'consent login'`

File: _source/_docs/api/resources/oidc.md (hunk 1 above)
Comment (mystiberry-okta) [reply _source/_docs/api/resources/oidc.md]: Ack.

File: _source/_docs/api/resources/oidc.md (hunk 2 above)
Comment (mystiberry-okta) [reply _source/_docs/api/resources/oidc.md]: ack.

**Pull request merged**